backend/src/routes/auth.js

```diff
@@ -74,7 +75,7 @@ router.get('/logout', async (req, res, next) => {
     if (req.query?.sessionExpired) {
       endpoint = '/session-expired';
```

Comment:
While we're at it I feel like we should fix up SessionExpired.vue as well. Bring the UI up to date and support BCeID/IDIR.

Reply:
Done!